analyzers/src/SonarAnalyzer.CSharp/Rules/AvoidExcessiveClassCoupling.cs

```diff
@@ -112,7 +112,8 @@ public sealed class AvoidExcessiveClassCoupling : ParameterLoadingDiagnosticAnal
                 SyntaxKind.ClassDeclaration,
                 SyntaxKind.StructDeclaration,
                 SyntaxKind.InterfaceDeclaration,
-                SyntaxKindEx.RecordClassDeclaration);
+                SyntaxKindEx.RecordClassDeclaration,
```

Comment:
Can you try to cleanup this class to match coding style? What I saw from a quick look:
* reorder members on top
* remove bunch of unneeded EOLs - useless empty lines and commands broke in half.